### PR TITLE
Fix display add event button for user

### PR DESCRIPTION
**Pull request by k-duchemin**

### CODE REVIEW

File: frontend/src/routes/index.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ export const AppRouter = () => {
         <PrivateRoute path="/clients"      component={ClientsPage} roles={['admin']} />
         <PrivateRoute path="/time_entries" component={TimeEntriesPage} />
         <PrivateRoute path="/events/:id/edit"  component={EditEventTemplatePage} roles={['admin']} />
-        <PrivateRoute path="/events/new"       component={NewEventTemplatePage} roles={['admin']}/>
+        <PrivateRoute path="/events/new"       component={NewEventTemplatePage}/>
```
Comment (Ven0mX) [inline frontend/src/routes/index.js:61]: why did u remove admin roles?

Comment (Ven0mX) [reply frontend/src/routes/index.js:61]: only an admin can create an event template?

**Review by Ven0mX — CHANGES_REQUESTED** [review]

File: frontend/src/routes/index.js (hunk 1 above)
Comment (k-duchemin) [reply frontend/src/routes/index.js:61]: the user can create a personal event for himself

Comment (Ven0mX) [reply frontend/src/routes/index.js:61]: yes but you have 2 endpoint
 - event template
 - user event
So i think you have to add a new path

Comment (k-duchemin) [reply frontend/src/routes/index.js:61]: Yeah you're right I close this PR for the moment and I create a new one when we take the feature for this case.

**Pull request closed without merging**